src/Tests/RegressionTests/RegressionTests.fs

```diff
@@ -11,14 +12,20 @@ let actualsPath = "../../../Actual"
 module Check =
 
     let fromFilename(output : ITestOutputHelper) (filename : string) =
-        let expected = new XLWorkbook(Path.Combine(expectedsPath, filename))
         let actual = new XLWorkbook(Path.Combine(actualsPath, filename))
+        let expected = new XLWorkbook(Path.Combine(expectedsPath, filename))
         Assert.Workbook.Equal(expected, actual, filename, output)    
 
 type Tests(output : ITestOutputHelper) =
+    do
+        if RuntimeInformation.IsOSPlatform(OSPlatform.Linux) then
+            LoadOptions.DefaultGraphicEngine <- ClosedXML.Graphics.DefaultGraphicEngine("DejaVu Sans")
 
-    [<Fact>]
-    member _.``RegressionTests`` () =
+    static member files =
         expectedsPath
         |> Directory.EnumerateFiles
-        |> Seq.iter (Path.GetFileName >> (Check.fromFilename output))
+        |> Seq.map (Path.GetFileName >> Array.singleton)
+
+    [<Theory>]
+    [<MemberData("files")>]
```

Comment:
Quality of life upgrade - with this each of the tests is displayed as a separate test case in test runners.